server/public/model/config.go

```diff
@@ -233,6 +233,8 @@ const (
 	PluginSettingsDefaultMarketplaceURL    = "https://api.integrations.mattermost.com"
 	PluginSettingsOldMarketplaceURL        = "https://marketplace.integrations.mattermost.com"
 
+	ComplianceExportDirectoryFormat    = "compliance-export-2006-01-02-15h04m"
+	ComplianceExportPath               = "export"
```

Comment:
@streamer45 It all depends on the customer's settings. 
If they're using `FileSettings.DriverName: "local"`, and `FileSettings.Directory: ./data/` (default), the compliance export will be put into `./data/export`. If `ExportSettings.Directory` is `./data/export` then it looks like it will delete them.

This PR wouldn't do anything to prevent that. Thinking about this, I think we should file a bug to fix the ExportSettings export_delete worker to raise an error and never run if the directories are the same. What do you think?

It doesn't /look/ like the export_delete worker cleans up S3, so we're safe there.

Reply:
Agreed, let's file a ticket. We should fix it and likely update our known issues docs. /cc @nab-77

Reply:
https://mattermost.atlassian.net/browse/MM-60888